src/goose/brokers/redis/console/pages/enqueued.clj

```diff
@@ -141,7 +141,9 @@
         validated-params (validate-get-jobs params)
         {:keys [queue] :as data} (data/enqueued-page-data (:redis-conn broker) validated-params)]
     (response/response (view "Enqueued" (assoc data :job-type :enqueued
-                                                    :base-path (prefix-route "/enqueued/queue/" queue)
+                                                    :base-path (if-let [q queue]
```

Comment:
> why?
If there are no queues, filter-request redirects to `/enqueued/queue/?filter-type=id&filter-value=1234` which is an invalid url. Instead if no queue is present redirection to `/enqueued?filter-type=id&filter-value=1234` makes a valid requeust.